integration-test/src/test/java/com/datastrato/gravitino/integration/test/catalog/hive/ProxyCatalogHiveIT.java

```diff
@@ -0,0 +1,252 @@
+    String comment = "comment";
+
+    catalog.asSchemas().createSchema(ident, comment, properties);
+    Database db = hiveClientPool.run(client -> client.getDatabase(schemaName));
+    Assertions.assertEquals(EXPECT_USER, db.getOwnerName());
+    Assertions.assertEquals(
+        EXPECT_USER, hdfs.getFileStatus(new Path(db.getLocationUri())).getOwner());
+    properties.put(
+        "location",
+        String.format(
+            "hdfs://%s:%d/user/hive/warehouse/%s.db",
+            containerSuite.getHiveContainer().getContainerIpAddress(),
+            HiveContainer.HDFS_DEFAULTFS_PORT,
+            anotherSchemaName.toLowerCase()));
+    Assertions.assertThrows(
+        RuntimeException.class,
+        () -> anotherCatalog.asSchemas().createSchema(anotherIdent, comment, properties));
+  }
+
+  @Test
+  public void testOperateTable() throws Exception {
+    Column[] columns = createColumns();
+    String schemaName = GravitinoITUtils.genRandomName(SCHEMA_PREFIX);
+    String tableName = GravitinoITUtils.genRandomName(TABLE_PREFIX);
+    String anotherTableName = GravitinoITUtils.genRandomName(TABLE_PREFIX);
+    NameIdentifier ident = NameIdentifier.of(METALAKE_NAME, CATALOG_NAME, schemaName);
+    NameIdentifier nameIdentifier =
+        NameIdentifier.of(METALAKE_NAME, CATALOG_NAME, schemaName, tableName);
+    NameIdentifier anotherNameIdentifier =
+        NameIdentifier.of(METALAKE_NAME, CATALOG_NAME, schemaName, anotherTableName);
+    Map<String, String> properties = Maps.newHashMap();
+    properties.put("key1", "val1");
+    properties.put("key2", "val2");
+    properties.put(
+        "location",
+        String.format(
+            "hdfs://%s:%d/user/hive/warehouse/%s.db",
+            containerSuite.getHiveContainer().getContainerIpAddress(),
+            HiveContainer.HDFS_DEFAULTFS_PORT,
+            schemaName.toLowerCase()));
+    String comment = "comment";
+    catalog.asSchemas().createSchema(ident, comment, properties);
+    Table createdTable =
+        catalog
+            .asTableCatalog()
+            .createTable(
+                nameIdentifier,
+                columns,
+                comment,
+                ImmutableMap.of(),
+                Partitioning.EMPTY_PARTITIONING);
+    String location = createdTable.properties().get("location");
+    Assertions.assertEquals(EXPECT_USER, hdfs.getFileStatus(new Path(location)).getOwner());
+    org.apache.hadoop.hive.metastore.api.Table hiveTab =
+        hiveClientPool.run(client -> client.getTable(schemaName, tableName));
+    Assertions.assertEquals(EXPECT_USER, hiveTab.getOwner());
+    Assertions.assertThrows(
+        RuntimeException.class,
+        () ->
+            anotherCatalog
+                .asTableCatalog()
+                .createTable(
+                    anotherNameIdentifier,
+                    columns,
+                    comment,
+                    ImmutableMap.of(),
+                    Partitioning.EMPTY_PARTITIONING));
```

Comment:
Also here.

Reply:
Please also add necessary white space and comments to the code blocks. You put everything together without any ws and comments, which is hard to review.

Reply:
Fixed.